examples/website/i3s-picking/src/components/drop-down.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from 'styled-components';
+import {Color, Font} from './styles';
+
+export const DropDownStyle = `
+  position: static;
```

Comment:
`static` doesn't seem right because it is a child component inside `ControlPanel`

Reply:
Removed